spaces/unicope/index.json

```
@@ -0,0 +1,17 @@
+{
+  "key": "unicope",
+  "name": "Unicope",
+  "chainId": 1,
+  "decimals": 18,
+  "symbol": "UNICOPE",
+  "defaultView": "core",
+  "address": "0xc5074D5964De8B8157BF6D4F4846dF9BC415FDdf",
+  "token": "0xc5074D5964De8B8157BF6D4F4846dF9BC415FDdf",
+  "core": [
+    "0xf8fE0383da886bdd425dd070f6e57Fd21eac1C81"
+  ],
+  "min": 1,
+  "invalid": [],
+      }]
+  ]
+}
```

There is an indentation problem on that file. Please clean it.
```suggestion
  "invalid": []
}
```

---

fixed